test/xpu/extended/run_test_with_skip.py

```diff
@@ -163,6 +163,12 @@
     "test_compare_cpu_std_mean_xpu_bfloat16",
     "test_compare_cpu_sub_xpu_float16",
     "test_compare_cpu_var_mean_xpu_bfloat16",
+
+    # Precision error.
+    # Mismatched elements: 2 / 125 (1.6%)
+    # Greatest absolute difference: 0.001953125 at index (2, 0, 0) (up to 0.001 allowed)
+    # Greatest relative difference: 0.007568359375 at index (2, 0, 0) (up to 0.001 allowed)
+    "test_compare_cpu_cumprod_xpu_bfloat16",
```

Comment:
Do we still need skip it? We can pass in recent pre-ci?

Reply:
> Do we still need skip it? We can pass in recent pre-ci?

I enabled this test case after adding missing "," in the pr, now the failure is stable occurrence